scripts/tests/test_filter_multimappers.py

```diff
@@ -244,7 +244,7 @@ def test_find_best_alignments_multimappers_nh(self, alns):
         output = find_best_alignments([alns[0], alns[1]], True)
 
         assert len(output) == 1
-        assert output[0] == alns[0]
+        assert output[0] == alns[1]
```

Comment:
The use of indices to refer to alignments (rather than names of a dictionary) makes me a bit nervous. It's very easy to make mistakes here, plus it's impossible to look at a test function and understand what's going on without cross-referencing how the alingments list is created.

So consider creating a dictionary instead of a list and give expressive names (here and everywhere else you make use of the `alns` list). Ideally, the expressive names should also contain